upstream-manifests/federation/0.0.10/federation.v0.0.10.clusterserviceversion.yaml

```diff
@@ -8,6 +8,367 @@ metadata:
   annotations:
     capabilities: Basic Install
     categories: "OpenShift Optional, Integration & Delivery"
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "clusterregistry.k8s.io/v1alpha1",
```

Comment:
The examples in this list should be in the set of CRDs provided by the operator. In this PR, the goal is to document the types specifically related to configuring resource federation: `Cluster`, `FederatedCluster`, and `FederatedTypeConfig`. The other examples you have in this PR that are for resources created by users doing `enable` won't actually correspond to something they get out of the box with this spin of the operator. It's likely that we'll have an option for users to get those resources from an operator in time and will need examples for them then, but for those resources should not be included in the examples here.